### PR TITLE
Add missing "static" modifier for constants

### PR DESCRIPTION
**Pull request by jukzi**
To reduce memory per instance

### CODE REVIEW

**Comment by jukzi** [comment]
@tjwatson please merge

**Comment by akurtakov** [comment]
Build failed with:
Failed to execute goal org.eclipse.tycho.extras:tycho-p2-extras-plugin:4.0.0-SNAPSHOT:compare-version-with-baselines (compare-attached-artifacts-with-release) on project org.eclipse.osgi: Only qualifier changed for (org.eclipse.osgi/3.18.400.v20230609-0920). Expected to have bigger x.y.z than what is available in baseline (3.18.400.v20230509-2241) -> [Help 1]

**Comment by jukzi** [comment]
unrelated? fails in CMCoordinationTestCase

File: bundles/org.eclipse.equinox.launcher/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.equinox.launcher;singleton:=true
-Bundle-Version: 1.6.400.qualifier
+Bundle-Version: 1.6.500.qualifier
```
Comment (tjwatson) [inline bundles/org.eclipse.equinox.launcher/META-INF/MANIFEST.MF:5]: Why is this version incremented.  I don't see any changes to the launcher.

File: bundles/org.eclipse.equinox.servletbridge/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: %bundleName
 Bundle-Vendor: %providerName
 Bundle-SymbolicName: org.eclipse.equinox.servletbridge;singleton:=true
-Bundle-Version: 1.6.200.qualifier
+Bundle-Version: 1.6.300.qualifier
```
Comment (tjwatson) [inline bundles/org.eclipse.equinox.servletbridge/META-INF/MANIFEST.MF:6]: I don't see any changes to the servletbridge.

Comment (jukzi) [reply bundles/org.eclipse.equinox.servletbridge/META-INF/MANIFEST.MF:6]: the build complained about it

File: bundles/org.eclipse.equinox.launcher/META-INF/MANIFEST.MF (hunk 1 above)
Comment (jukzi) [reply bundles/org.eclipse.equinox.launcher/META-INF/MANIFEST.MF:5]: the build complained about it

Comment (HannesWell) [reply bundles/org.eclipse.equinox.launcher/META-INF/MANIFEST.MF:5]: Can you please try again without the extra version bumps?

Comment (jukzi) [reply bundles/org.eclipse.equinox.launcher/META-INF/MANIFEST.MF]: ok, removed it now it works.

File: bundles/org.eclipse.equinox.servletbridge/META-INF/MANIFEST.MF (hunk 2 above)
Comment (jukzi) [reply bundles/org.eclipse.equinox.servletbridge/META-INF/MANIFEST.MF]: removed it

**Review by HannesWell — APPROVED** [review]
Looks good now.
Thanks!

**Pull request merged**